src/libraries/System.Text.RegularExpressions/tests/RegexReductionTests.cs

```diff
@@ -6,7 +6,8 @@
 
 namespace System.Text.RegularExpressions.Tests
 {
-    [ActiveIssue("https://github.com/dotnet/runtime/issues/64300", TestPlatforms.Browser)]
+    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Many of these optimizations don't exist in .NET Framework.")]
+    [SkipOnPlatform(TestPlatforms.Browser, "Internal members accessed via reflection may have been trimmed away")]
```

Comment:
ideally we'd be using `PlatformDetection.IsBuiltWithAggressiveTrimming` instead so this will automatically work when turning on trimming for other platforms.

Reply:
I wasn't aware that existed.